drivers/power_domain/power_domain_psl.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_npcx_power_domain_psl
+
+#include <kernel.h>
+#include <drivers/gpio.h>
+#include <drivers/pinctrl.h>
+#include <pm/device.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(power_domain_psl, LOG_LEVEL_INF);
+
+struct npcx_psl_config {
+	struct gpio_dt_spec enable;
+	/* PSL_IN pinmux configuration */
+	const struct pinctrl_dev_config *pcfg;
+};
+
+struct npcx_psl_data {
+	int event;
+};
+
+const char *actions[] = {
+	[PM_DEVICE_ACTION_TURN_OFF] = "TURN OFF"
+};
+
```

Comment:
This looks unused? Were you meaning to use it in npcx_psl_pm_action()? Should probably be static if that's the case.

Reply:
Yes, it is. Will remove the other unused variables later.

Reply:
Done.